compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -616,7 +618,11 @@ cmdClean =
     info (helper <*> cmd) $
     progDesc "Remove Daml project build artifacts" <> fullDesc
   where
-    cmd = execClean <$> projectOpts "daml clean"
+    cmd = execClean
+            <$> projectOpts "daml clean"
+            <*> enableMultiPackageOpt
+            <*> multiPackageLocationOpt
+            <*> multiPackageCleanAllOpt
```

Comment:
I think you'll need to change the `Clean` entry in `cmdUseDamlYamlArgs` (near the very bottom of this file) to `True`, otherwise `daml clean` won't pick up the multi-package flags specified in `daml.yaml`

Reply:
hm but reading the comment on `multiPackageBuildEffect`, I see that we don't demand a `daml.yaml` file next to `multi-package.yaml`. That means that the only way to run this is by explicitly passing `enable-multi-package=yes`, i.e. `daml clean --all --enable-multi-package=yes`. If that's the intention, there's nothing to do

Reply:
Though, perhaps specifying the `--multi-package-path` in those args would be nice, i might put up a ticket to change that for clean and build